.github/workflows/rust_checks.yml

```diff
@@ -0,0 +1,59 @@
+name: Rust Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+    - name: Run tests
+      run: |
+        cd .. # Move up from the .github directory to find the workspace root
+        cargo test --all
```

Comment:
All jobs are still failing, but this time with a little different error message:
```
error: could not find `Cargo.toml` in `/home/runner/work/.github` or any parent directory
```